docs/kit/logistics/kit-testing.md

```diff
@@ -0,0 +1,221 @@
+# Kit Testing
```

Comment:
Issue: This doesn't include physical condition testing. Could the opening be updated to mention that,.

Reply:
We do now.